### PR TITLE
Adds pip installation support

### PR DESCRIPTION
**Pull request by PythooonUser**
## Summary
Adds support for a local `pip` installation of the repo. Small changes in the repo structure where necessary in order to make this happen (based on my understanding of package installation).

### CODE REVIEW

File: twitchobserver/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from twitchobserver.twitchobserver import TwitchChatObserver, TwitchChatEvent
```
Comment (JoshuaSkelly) [inline twitchobserver/__init__.py]: This style of import breaks Python2 compatibility.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,16 @@ Turn Twitch chatter into Python events.
 
 ## Usage
 
-### 1. Create an Observer
+### 1. Installation
+
+To install twitch-observer you need to download this GitHub repo and call `pip` in your shell of choice:
```
Comment (JoshuaSkelly) [inline README.md]: Maybe you don't have to be so explicit here? Our target audience is developers. Perhaps something like:
> Clone this repo and run:
> ```pip install .```

This is only a suggestion, I'd like to hear your thoughts.

File: twitchobserver/__init__.py (hunk 1 above)
Comment (PythooonUser) [inline twitchobserver/__init__.py]: Updated the import statement.
Tested with Ubuntu 16.04.2 LTS, pip (8.1.1, Python 2.7) and pip3 (8.1.1, Python 3.5).
Tested with Windows 10, pip (9.0.1, Python 3.5).

File: README.md (hunk 2 above)
Comment (PythooonUser) [inline README.md]: You are right, for the target audience this is to specific.
I also added a note for PyPI support in the future.

**Comment by PythooonUser** [comment]
I made those changes. Is there something else/new to change?

**Comment by JoshuaSkelly** [comment]
Nope! Looks good to me! 👍

**Pull request merged**